web/src/components/netflow-record/record-field.tsx

```diff
@@ -122,8 +124,12 @@ export const RecordField: React.FC<{
   };
 
   const dateTimeContent = (date: Date | undefined) => {
-    const dateText = date?.toDateString() || emptyText();
-    const timeText = date?.toLocaleTimeString() || emptyText();
+    if (!date) {
+      return emptyText();
+    }
+
+    const dateText = getFormattedDate(date, longDates ? 'ddd ll' : 'MM/DD/YYYY');
```

Comment:
that's the american format? Can we have it based on browser locale? (if not painful)

Reply:
If you don't mind using moment.js, then do something like this so it considers locale and handles milliseconds.  Example:  
  
`const shortFormat = moment('2022-10-11 08:25:01.123').format('l hh:mm:ss.SSS');`  
`const longFormat = moment('2022-10-11 08:25:01.123').format('ddd ll h:mm:ss.SSS A');`  
  
`moment` accepts many formats for the date/time argument or leave it empty if you want the time now.

Reply:
Moved to Intl.DateTimeFormat